### PR TITLE
107-add-project-documentation

### LINKED ISSUE #107: Add project documentation
- [ ] Invasive Insects
- [ ] Poultry and Red Meat Condemnations

### PR DESCRIPTION
**Pull request by RussellJimmies**
#107

### CODE REVIEW

**Review by ClemZ31 — COMMENTED** [review]
LGTM ! 
If you could just review the french part before we merged it.

**Review by ClemZ31 — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/current/projects/poultryredmeatcond/introduction.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
-# Introduction to Poultry and Red Meat Condemnations Project
+# Projet "Poultry and Red Meat Condemnations"
+
+![texte alternatif](CAHSS_Dashboard.png)
+
+## Vue d'ensemble
+
+L'équipe de Visualisation de Données de l'ACIA extrait des données des rapports
+de [condamnations de la volaille et des viandes rouges dans les établissements
+enregistrés au fédéral
```
Comment (RussellJimmies) [inline i18n/fr/docusaurus-plugin-content-docs/current/projects/poultryredmeatcond/introduction.md]:
```suggestion
condamnations de la volaille et du bétail dans les établissements sous inspection fédérale
```

File: i18n/fr/docusaurus-plugin-content-docs/current/projects/invasiveinsects/introduction.md [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+s'avère être un outil puissant pour relever ce défi.
+
+### Solution
+
+Nous tirons parti du savoir de la foule par le biais du service de réseau social
+iNaturalist. Ce Service de Réseau Social (SRS) consiste en naturalistes,
+citoyens scientifiques et biologistes qui partagent des observations de la
+biodiversité à travers le monde.
+
+Nous générons des rapports hebdomadaires complets à partir de ces observations
```
Comment (RussellJimmies) [inline i18n/fr/docusaurus-plugin-content-docs/current/projects/invasiveinsects/introduction.md]: Quotidiens au lieu de hebdomadaires

File: docs/projects/invasiveinsects/introduction.md [hunk 3]
```diff
@@ -1,4 +1,35 @@
...
+challenge.
+
+### Solution
+
+We leverage knowledge of the crowd through the iNaturalist social networking
+service. This Social Networking Service (SNS) consists of naturalists, citizen
+scientists, and biologists who share observations of biodiversity across the
+globe.
+
+We generate weekly comprehensive reports from these crowd-sourced observations
```
Comment (RussellJimmies) [inline docs/projects/invasiveinsects/introduction.md]: daily instead of weekly

**Pull request merged**